examples/compiled/layer_text_heatmap.vg.json

```diff
@@ -123,7 +123,7 @@
       "direction": "horizontal",
       "gradientLength": 120,
       "fill": "color",
-      "title": "Count of Records"
+      "title": "num_cars"
```

Comment:
The original version of this example made a direct reference to the `count_*` field generated by a count aggregation in the embedding. In order to make it so that the example wasn't dependent on the automatically generated name for the transformed field I extracted the transformation.

I could keep the name consistent by using an explicit title in the encoding.

Reply:
Yes, please make it explicit.